xds/internal/server/listener_wrapper.go

```diff
@@ -302,7 +292,7 @@ func (l *listenerWrapper) Accept() (net.Conn, error) {
 		// tradeoff for simplicity.
 		vhswi, err := fc.ConstructUsableRouteConfiguration(rc)
 		if err != nil {
-			l.logger.Warningf("route configuration construction: %v", err)
+			l.logger.Warningf("Constructing usable route configuration: %v", err)
```

Comment:
Shouldn't this say `Error from ...` or something like that?

Reply:
I have trying to follow an approach I read on go-nuts@ at some point in time. I'm not able to locate it now. But the idea is to use concise error/log strings such that we avoid filler words like "failed", "error" etc, instead say "what we are doing" followed by the error string returned by the called function.

So, this might end up looking something like:
`Constructing usable route configuration: constructing route matcher: illegal route: missing path_matcher`

If we use filler words this might look like:
`Failed to construct usable route configuration: failed to construct route matcher: illegal route: missing path_matcher`

Reply:
Maybe in the case of error strings this is okay, but in the case of log messages, might not be the same thing?  I think if you're returning an error, then saying "error" at the start of the string is where you'd run into those composition problems.  But for log messages, you aren't returning them, so they won't ever be concatenated with other errors.

Reply:
Fair point. Done.